ArduPlane/config.h

```diff
@@ -158,18 +156,15 @@
 //////////////////////////////////////////////////////////////////////////////
 // Autopilot control limits
 //
-#ifndef HEAD_MAX
```

Comment:
I wonder if we should leave some errors to save people defining these in the hwdef. EG:
```
#ifdef HEAD_MAX
    #error HEAD_MAX deprecated use ROLL_LIMIT_DEG
#endif
```

Reply:
I don't think it is worth it as those config.h changes are not used as far as I know, defaults.parm is the current method